### PR TITLE
PD-406: Add ui-colors.less file

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

- Palette package now includes palette.less file
- Added file_loader module (so that palette.less is included in dist)

🎟 _Jira ticket:_ [PD-406](https://jira.mongodb.org/browse/PD-406)

## 🛠 Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,5 +122,8 @@
     "packages": [
       "packages/*"
     ]
+  },
+  "dependencies": {
+    "file-loader": "^4.2.0"
```
Comment (DesignerDave) [inline package.json]: PeerDependencies

File: packages/palette/src/uiColors.less [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/* LeafyGreen Color Palette */
```
Comment (DesignerDave) [inline packages/palette/src/uiColors.less]: This comment is unnecessary.

File: package.json (hunk 1 above)
Comment (brookescarlett) [reply package.json]: 👍

File: packages/palette/src/uiColors.less (hunk 2 above)
Comment (brookescarlett) [reply packages/palette/src/uiColors.less]: 👍

**Pull request merged**